resources/translations/fi.edn

```diff
@@ -281,6 +281,9 @@
           :decision-requested {:subject "(%3) Päätöspyyntö"
                                :message "Hyvä %1,\n\n%2 on pyytänyt sinulta päätöstä käyttäjän %4 hakemuksesta %3.\n\nVoit tarkastella hakemusta osoitteessa %6"}
           :footer "\n\nTämä on automaattinen viesti. Älä vastaa."
+          :handler-reminder {:application "%1, lähettänyt %2"
+                             :message "Hyvä %1,\n\nSeuraavat hakemukset ovat vaiheessa ja voivat vaatia toimiasi:\n\n%2\n\nVoit tarkastella hakemuksia osoitteessa %3"
+                             :subject "Hakemuksia vaiheessa"}
```

Comment:
Similarly, I think "Hakemusten käsittely kesken" would be more formal here, although admittedly a bit long. But I'd prefer that in any case.

Reply:
Hmm, after thinking a bit, I'm not happy with the above proposal either -- how about "Keskeneräiset hakemukset"? I think it would have a close correspondence with the English translation ("Applications in progress").

Reply:
Okay, this came from Leena: "Avoimet hakemukset", probably the best of these proposals.